src/test/java/teammates/test/cases/BaseComponentTestCase.java

```diff
@@ -137,9 +137,20 @@ protected static void verifyPresentInDatastore(FeedbackResponseAttributes expect
         if (wildcardId) {
             expected.setId(actual.getId());
         }
+        
+        setEqualTimestampBetweenExpectedFeedbackResponseAttributesAndActual(expected, actual);
         assertEquals(gson.toJson(expected), gson.toJson(actual));
     }
     
+    private static void setEqualTimestampBetweenExpectedFeedbackResponseAttributesAndActual(FeedbackResponseAttributes expected,
```

Comment:
This method name is too specific to how the method is used. On its own, the just equalizes the time stamp, and should be named/described as such. e.g. `equalizeTimeStamps`